### PR TITLE
added base test

### PR DESCRIPTION
**Pull request by k2shah**
added examples for a basic test and the cmake target

### CODE REVIEW

File: mslquad/test/test_hello.cpp [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  int n = 1;
+  ASSERT_EQ(n, 1);
+}
+
+TEST(TestSuite, testFloat){
+  float a,b;
+  a = 1;
+  b = .5;
+  ASSERT_EQ(a+b, 1.5);
+}
```
Comment (exoticDFT) [inline mslquad/test/test_hello.cpp:27]: Should add a new line here between the main and test function.

**Review by exoticDFT — COMMENTED** [review]
Things look good. I tried to do a bit more digging about using the XML to display the results more visually in the PR. It seems this may require creating yet another account or host our own server somewhere to process and display the results. Doesn't seem worth it at the moment.

My only concern with the PR is we're getting rid of the VRPN client script in a completely unrelated titled PR. It might be worth waiting to remove that specific file when it's more directly related to the PR, or we should try to explicitly mention this so it doesn't seem to have randomly disappeared.

File: mslquad/test/test_hello.cpp (hunk 1 above)
Comment (k2shah) [reply mslquad/test/test_hello.cpp:27]: done

**Comment by k2shah** [comment]
@exoticDFT we good? if this passes should i merge?

**Review by exoticDFT — APPROVED** [review]
Looks great!

**Pull request merged**